### ISSUE TITLE
Handling the error in RepositoryMining when the repository is empty

### ISSUE THREAD

**Issue by mohmehmo** [ishepard/pydriller#154]
Hello,
Firstly, thanks a lot for your great tool. I would like to ask you to add a feature to handle the error while reviewing the commits of a repository using _RepositoryMining_, when the repository is empty. For instance, when I use the following code, an error is raised which results in an empty repository. 
```python
commits = RepositoryMining("repo").traverse_commits()
for commit in commits:
     msg = commit.msg
```

**Comment by ishepard** [comment]
Hi @mohmehmo!
Yeah you are right, if the repository is empty we should probably just return immediately, without raising the error.
Would you be available to look into it and create a PR? 😃 

Thank you!

**Comment by mohmehmo** [comment]
Hi @ishepard 
Yes, I will try to work on it on weekend.

**Comment by ishepard** [comment]
@mohmehmo any news on this? Just checking if this can be included in the Pydriller 2.0 release 😄
